sockeye/decoder.py

```diff
@@ -363,6 +363,8 @@ def decode(self,
         lexical_biases = []
 
         self.rnn.reset()
+        for cell in self.rnn._cells:
```

Comment:
As noted before, this should ideally be fixed by MXNet so that we don't have to access private members.